dozer/cogs/moderation.py

```diff
@@ -705,6 +719,25 @@ async def nmconfig(self, ctx, channel_mention: discord.TextChannel, role: discor
     so if someone types "I have read the rules and regulations" it assigns them the Member role. 
     """
 
+    @command()
+    @has_permissions(administrator=True)
+    async def nmpurgeconfig(self, ctx, role: discord.Role, days: int):
+        """Sets the config for the new members purge"""
+        config = await NewMemPurgeConfig.get_by(guild_id=ctx.guild.id)
+        if len(config) != 0:
+            config = config[0]
+            config.member_role = role.id
+            config.days = days
+        else:
```

Comment:
Could this be simplified to just
`config = NewMemPurgeConfig(guild_id=ctx.guild.id, member_role=role.id, days=days)`
`await config.update_or_add()`

Reply:
Yup, that was an oversight